### PR TITLE
complete Desktop view

### PR DESCRIPTION
**Pull request by saied2035**
## **In** this branch I made the desktop version of the project and made it resopnsive to the desktop&mobile

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -351,3 +377,185 @@ textarea {
   margin-block-end: 12rem;
   margin-block-start: 0;
 }
+
+/* desktop view */
+@media only screen and (min-width: 800px) {
```
Comment (mmsesay) [inline index.css]: - [x] You did well setting the media query for your website. However, this min-width value `800px` is a bit smaller. For your website to be responsive on other screens, I will suggest you increase the min-width value to `992px`.

File: index.html [hunk 2]
```diff
@@ -123,17 +132,19 @@ <h1 id="works-title">My Recent Works</h1>
...
+
     <section id="About-section">
     	   <h1 id="About-title">About me</h1>
 		   <p id="About-text">
 	    	   	I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded
 	    	   	, don’t hestiate to contact me.
     	   </p>
     	   <img id="About-img2" alt="About image 2" src="./images/aboutimg2.png">
-    	   <button type="submit" id="About-button">Get My Resume</button>
+    	   <button type="button" id="About-button">Get My Resume</button>
```
Comment (mmsesay) [inline index.html:146]: - [x] I think that the about section contents are well placed. But I found out that your design has a large space between the `p` tag and the `button`. I believe you can fix that. Try checking if you apply some padding in that section. 

Your Design | Figma Design |
:---------------:|:-------------------:|
e1 |e3

File: index.html [hunk 3]
```diff
@@ -123,17 +132,19 @@ <h1 id="works-title">My Recent Works</h1>
...
+    	   <div class="categories">
```
Comment (mmsesay) [inline index.html:147]: - [x] These section contents are well placed. But I found out that your design has a smaller space between the ending cards and the contact form section. I believe you can fix that by adding the spacing to match with the Figma design.
Also, take note of the input focus colours.

Your Design | 
:---------------:|
 e2

Figma Design |
:------------------:|
e4

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @saied2035 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi @saied2035👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 




Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — COMMENTED** [review]
Hi @saied2035 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a closed Pull request.

Please ensure that you open a new pull request with the branch that has a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section')

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**